internal/generate/ci/generate_test.go

```diff
@@ -10,73 +10,20 @@ import (
 	"github.com/commitdev/commit0/internal/templator"
 )
 
-func TestGenerateJenkins(t *testing.T) {
```

Comment:
I guess these cases are covered by the integration tests now anyway, plus the logic inside of the CI generator is much simpler now.